src/main/java/io/strimzi/kafka/bridge/mqtt/mapper/MappingResult.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.mapper;
+
+/**
+ * Represents the result of a mapping operation.
+ * It contains the mapped kafka topic and the kafka key.
+ */
+public class MappingResult {
+    private String kafkaTopic;
+    private String kafkaKey;
```

Comment:
I would make these two final so that they can be inizialized just in the constructor and removing the setters from this class.

Reply:
fixed sensei.